types/mapcmd/map.go

```diff
@@ -17,19 +17,40 @@ package mapcmd
 
 import (
 	"context"
+	"errors"
```

Comment:
This file seems to have quite a lot of changes not related to the PR. Are they intentional?

Reply:
Yes, as we discussed. Those are related to PR.